packages/react/toolbar/src/Toolbar.test.tsx

```diff
@@ -1,8 +1,6 @@
-import React from 'react';
```

Comment:
Had to please the pre-push hooks (how has this stuff worked before?)

Other changes in this file are due to my formatting on save